WcaOnRails/db/migrate/20231103102744_add_region_to_user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddRegionToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :users, :region, foreign_key: { to_table: :user_groups }
```

Comment:
Can you change this so it is added after the `delegate_status` column?

Reply:
Done.